docs/_docs/interface.md

```diff
@@ -8,29 +8,28 @@ toc: false
 
 ## Home
 
-When you browse to the root of your registry, it's a fairly expected interface with
-links to collections and documentation. Here is what we see for the default "Tacosaurus Computing Center"
+When you browse to your registry's root, it's a reasonably expected interface with
+links to collections and documentation. Here is what we see for the default "Tacosaurus Computing Center".
```

Comment:
This period should be on the inside of the quote.

Reply:
make that a double shot